instructions/FOSS_staging_list.pl

```diff
@@ -182,7 +182,7 @@
       "dir"         => "zm-help",
       "ant_targets" => undef,
       "stage_cmd"   => sub {
-         SysExec("cp -f -r ../zm-help $CFG{BUILD_DIR}");
+         SysExec("(cd .. && rsync -az --relative --exclude '.git' zm-help $CFG{BUILD_DIR}/)");
```

Comment:
just curious as there is no description, how are .git making to build and what issue is it causing

Reply:
updated commit message to provide more information, basically .git directories were giving some warnings in RHEL7 while upgrading from 9 to 10
this change will make sure git directories are not part of build and we should never ship git directories in build